guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -43,14 +50,68 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # subscription-manager repos --enable=rhel-7-server-extras-rpms
 ----
+ifdef::satellite[]
+.. Enable the `rhel-7-server-rpms` repository:
++
+----
+# subscription-manager repos --enable=rhel-7-server-rpms
+----
+.. Enable the `rhel-7-server-satellite-{ProjectVersion}-rpm` repository:
++
+[options="nowrap" subs="quotes, attributes"]
+----
+# subscription-manager repos --enable=rhel-7-server-satellite-{ProjectVersion}-rpm
+----
++
+These repositories can come from a locally mounted ISO. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html-single/installing_satellite_server_in_a_disconnected_network_environment/index#configuring-the-base-operating-system-with-offline-repositories-in-rhel-7_satellite[Configuring the Base Operating System with Offline Repositories in RHEL 7] in _Installing Satellite Server in a Disconnected Network Environment_.
+endif::[]
 
 . Install required packages:
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 +
 ----
 # {package-install-project} leapp leapp-repository
 ----
+ifdef::satellite[]
++
+. For Leapp to perform the upgrade in a disconnected environment, the metadata should be downloaded and extracted manually, as described in https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+
+
++
+. Set up the following repositories to perform the upgrade in a disconnected environment:
+.. `/etc/yum.repos.d/rhel8.repo`:
++
+----
+[BaseOS]
+name=rhel-8-for-x86_64-baseos-rpms
+baseurl=http://server.example.com/rhel8/BaseOS/
+
+[AppStream]
+name=rhel-8-for-x86_64-appstream-rpms
+baseurl=http://server.example.com/rhel8/AppStream/
+----
+
++
+.. `/etc/yum.repos.d/satellite.repo:`
+[options="nowrap", subs="+quotes,verbatim,attributes"]
++
+----
+[satellite-6.11-for-rhel-8-x86_64-rpms]
+name=satellite-6.11-for-rhel-8-x86_64-rpms
+baseurl=http://server.example.com/sat6/Satellite/
+
+[satellite-maintenance-6.11-for-rhel-8-x86_64-rpms]
+name=satellite-maintenance-6.11-for-rhel-8-x86_64-rpms
+baseurl=http://server.example.com/sat6/Maintenance/
+----
 
+. Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO.
++
+[NOTE]
+====
+These repositories are delivered over the network from a different machine.
+====
```

Comment:
```suggestion
+
[NOTE]
====
Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO but must be delivered over the network from a different machine.
====
```